R/utils.R

```diff
@@ -190,3 +196,18 @@ download_method <- function() {
   if (is.na(capabilities()["libcurl"])) "internal" else "libcurl"
 }
 
+needs_packages <- function(pkgs) {
+  for (pkg in pkgs) {
+    if (!requireNamespace(pkg, quietly = TRUE)) {
+      stop("The ", pkg, " package is needed for this addin to work.")
```

Comment:
I would prefer the addin to tell me all packages that are missing at once. I had not installed shinyjs and shinywidget, the first time it only told me about shinyjs.

Furthermore I'd prefer `call.=FALSE`

Reply:
Yeah. Maybe it could even offer to install them? It is not really ideal that they are all suggested, maybe we could just import all of them...